docs/dev_101_index.md

```diff
@@ -1,49 +0,0 @@
----
```

Comment:
`/docs/dev_101_index /docs/dev_101_hass` should be added to `static/_redirects` to ensure links will continue to work.